Bindings/Java/OpenSimJNI/OpenSimContext.cpp

```diff
@@ -237,6 +237,11 @@ void OpenSimContext::setLocation(PathPoint& mp, int i, double d) {
     recreateSystemKeepStage();
 }
 
+void OpenSimContext::setLocation(PathPoint& mp, const SimTK::Vec3& newLocation) {
+    mp.setLocation(newLocation);
+    recreateSystemKeepStage();
```

Comment:
This seems consistent with property editing functionality on `OpenSimContext`. :+1: